package.json

```diff
@@ -55,7 +55,7 @@
   "dependencies": {
     "chalk": "^1.1.3",
     "commander": "^2.9.0",
-    "directory-tree": "^1.2.1",
+    "directory-tree": "^2.0.0",
```

Comment:
This is the biggest change. My PR for this plugin got merged so we can now use a regex for comparisons.